README.md

```diff
@@ -30,23 +30,92 @@ npm install --save ipfsd-ctl
 
 IPFS daemons are already easy to start and stop, but this module is here to do it from JavaScript itself.
 
+### Local node
+
 ```js
 // Start a disposable node, and get access to the api
-// print the node id, and kill the temporary daemon
+// print the node id, and stop the temporary daemon
 
 // IPFS_PATH will point to /tmp/ipfs_***** and will be
 // cleaned up when the process exits.
 
-var ipfsd = require('ipfsd-ctl')
+const factory = require('ipfsd-ctl')
+const localController = factory.localController
 
-ipfsd.disposableApi(function (err, ipfs) {
+localController.spawn(function (err, ipfsd) {
+  const ipfs = ipfsd.ctl
+  const node = ipfsd.ctrl
   ipfs.id(function (err, id) {
     console.log(id)
-    process.exit()
+    node.stopDaemon()
   })
 })
 ```
 
+### Remote node
+
+```js
+// Start a remote disposable node, and get access to the api
+// print the node id, and stop the temporary daemon
+
+// IPFS_PATH will point to /tmp/ipfs_***** and will be
+// cleaned up when the process exits.
+
+const ipfsd = require('ipfsd-ctl')
+const server = ipfsd.server
+
+server.start((err) => {
+  if (err) {
+    throw err
+  }
+  
+  const remoteController = ipfsd.remoteController(port || 9999)
+  remoteController.spawn(function (err, controller) {
+    const ipfs = controller.ctl
+    const node = controller.ctrl
+    ipfs.id(function (err, id) {
+      console.log(id)
+      node.stopDaemon()
+      server.stop()
+    })
+  })  
+})
+```
+
+It's also possible to start the server from `.aegir` `pre` and `post` hooks. For reference take a look at the `.aegir` file in this repository.
+
+
+## API
+
```

Comment:
The API should be divided into 3 sections:

- Daemon Factory
- IPFS Client
- IPFS Controller

Each with its respective methods.